radiation/fortran/radsw/radsw_main2.F

```diff
@@ -914,6 +914,7 @@ subroutine swrad                                                  &
       lab_do_ipt : do ipt = 1, NDAY
 
         j1 = idxday(ipt)
+        write(*,*) "j1 =",j1 
```

Comment:
was this intended for debugging only, or prefer to always report it? Totally fine either way, just checking if it's intentional.

Reply:
Ah thanks, this should have been taken out